plugins/easyimage/plugin.js

```diff
@@ -504,10 +504,50 @@
 		return !CKEDITOR.env.ie || CKEDITOR.env.version >= 11;
 	}
 
+	function addUploadButtonToToolbar( editor ) {
+		var hiddenUploadElement;
+		if ( !editor._.easyImageHiddenUploadElement ) {
+			// Element is not attached to DOM, but still it might be `virtually` clicked.
+			hiddenUploadElement = CKEDITOR.dom.element.createFromHtml( '<input data-cke-easyimage-hidden-upload="1" style="display:none;" type="file" accept="image/*" tabindex="-1" multiple="true">' );
```

Comment:
Most of these attributes are not needed. `[type]`, `[accept]` and `[multiple]` should be enough.

What's more, value for [`[multiple]` attribute](https://html.spec.whatwg.org/#the-multiple-attribute) is incorrect. It's a boolean attribute, so it shouldn't have a value or (if we prefer XHTML style) it's value should be set to the attribute's name (`[multiple=multiple]`).